doc/arch/014-reporting-etl-pipeline.md

```diff
@@ -0,0 +1,150 @@
+# Reporting ETL Pipeline
+
+## Context
+
+Simple's dashboard reports are currently generated by queries on the application database's tables. This has several
+disadvantages as we start to scale.
+
+### 1. Complexity
+
+Simple's application schema is optimized for application transactions like registering patients, recording BPs,
+updating patient info, or tracking overdue patients. As such, reporting queries grow complex. Asking questions like
+"latest BP in a month", "prescribed drugs as of a date", "current step of treatment protocol" involve complex clauses
+in the SQL queries.
+
+This complexity has a tangible impact on the quality of our reports. We risk data inconsistency as complex queries are
+more error-prone when reused in multiple places.
+
+
+### 2. Coupling
```

Comment:
I feel the first point is talking about complexity introduced through coupling of the transactional schema with the reporting schema. 

Would it be better to keep this second one about performance?